src/env.cr

```diff
@@ -52,6 +52,29 @@ module ENV
     !!LibC.getenv(key)
   end
 
+  # Fetch an environment variable corresponding to a key. Throws an exception
+  # if the key does not exist. If a second argument or a block is provided then
+  # this value is returned instead of throwing the exception.
+  #
+  def self.fetch(key)
+    fetch(key) do
+      if block = @@block
+        block.call(self, key)
+      else
+        raise KeyError.new "Missing ENV key: #{key.inspect}"
+      end
+    end
+  end
```

Comment:
I thought about that, to reduce aliases. But `fetch` could be the method and `[]` a convenience accesor.

Ni strong opinion, except that `fetch(key, default)` is nice. But then, no `fetch(key)` may be weird.